docs/pages/api-reference/operators.md

```diff
@@ -130,9 +130,24 @@ Fennel does not allow you to drop keys or timestamp columns from a dataset.
 
 <pre snippet="api-reference/operators_ref#drop"></pre>
 
+
+### Dropnull
+
+Fennel allows you to drop null values from columns of a dataset using the `dropnull` operator.
```

Comment:
We should mention that this also changes the type of the column from `Option[T]` to `T`.